### PR TITLE
Update Readme.md file with link to Contribution guide

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
- [x]  Added the License agreement
- [x] Added the more efficient links to the issues so that they do not break when they are forked.
- [x] Added the year and the owner name in the license agreement 
- [x] Fixes issue #753

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -396,13 +396,14 @@ Finally, we can use the `make kind-run` target to run the current version of kub
 Every time you want to test a change, you'll need to rebuild the docker image and push it to cluster before running it again. ( `make build-docker kind-push kind-run` )
 
 ## Contributing
+Please encourage and support all the contribution to this project. Kindly read [Contributing.md](CONTRIBUTING.md) before contributing. Some instructions for the common contributions are stated below.
```
Comment (lizrice) [inline README.md:399]:
```suggestion
Kindly read [Contributing.md](CONTRIBUTING.md) before contributing. Some instructions for the common contributions are stated below.
```

Deleted the first sentence as it's not clear who it's aimed at (contributors or maintainers?) I do like the addition of a link to the Contributing page.

**Review by lizrice — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -396,6 +396,7 @@ Finally, we can use the `make kind-run` target to run the current version of kub
 Every time you want to test a change, you'll need to rebuild the docker image and push it to cluster before running it again. ( `make build-docker kind-push kind-run` )
 
 ## Contributing
+Kindly read [Contributing.md](CONTRIBUTING.md) before contributing. Some instructions for the common contributions are stated below.
```
Comment (kaiwalyakoparkar) [inline README.md:399]: @lizrice  I have made changes as requested. Kindly review these changes.

File: README.md [hunk 3]
```diff
@@ -426,3 +427,7 @@ We welcome pull requests!
 - Your PR is more likely to be accepted if it includes tests. (We have not historically been very strict about tests, but we would like to improve this!).
 - You're welcome to submit a draft PR if you would like early feedback on an idea or an approach.
 - Happy coding!
+
+## License Agreement
+
+This entire project comes with a Apache License, Version 2.0. Kindly read it [here](LICENSE)
```
Comment (lizrice) [inline README.md:433]: Please remove this, we don't need to duplicate the information that's already covered by License

**Review by lizrice — CHANGES_REQUESTED** [review]
Thanks for the changes, this is nearly there, but please note we REALLY don't want any changes to do with licensing. At all!

**Review by lizrice — APPROVED** [review]
lgtm

**Pull request merged**